kokoro/config/test/image_lists_3p_override.gcl

```diff
@@ -0,0 +1,85 @@
+import 'image_lists.gcl' as image_lists
+
```

Comment:
I think I would prefer if we could make 2 things happen here:

*  put presubmits in this file as well, just like image_lists.gcl
*  make this file hold all distros in one tuple/map instead of having it hold just the overrides and having to fall back to image_lists.gcl if it isn't found. This will make it a lot easier to refer to this file from the various presubmit files (such as https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/third_party_apps/bullseye_x86_64.gcl)

The solution I have in mind will also dodge an issue that I suspect will happen when doing `image_lists.get(_distro_arch, 'invalid_distro')`: I suspect that `.get()` won't work because `image_lists` is an imported file and not a proper tuple. I mean maybe `.get` will work, I dunno.

But anyway, the solution I have in mind is to put everything in both files into tuples, called I dunno `distros`:

```
distros = {
  bookworm_aarch64 = distro {
    ...
  }
  mantic_x86_64 = distro {
    ...
  }
```

Then in image_lists_3p.gcl, we would write:

```
distros = image_lists.distros {
  mantic_x86_64 {
    presubmit = [
      'foo'
    ] ,
     release = [
      'ubuntu-2310-amd64',
    ]
  }
  mantic_aarch64 {
    ...
  }
}
```

Then `.get` is guaranteed to work because `distros` is a tuple, and it allows us to refer to `image_lists_3p.distros` without needing to fall back to `image_lists`.

Reply:
I'm going to try doing something else entirely here, and use the `presubmit` distros for 3p tests.